systemtest/src/main/java/io/strimzi/systemtest/security/SystemtestCertManager.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.security;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+import static java.util.Arrays.asList;
+
+public class SystemtestCertManager {
+
+    private static final String KAFKA_CERT_FILE_PATH = "/opt/kafka/broker-certs/";
+    private static final String ZK_CERT_FILE_PATH = "/etc/tls-sidecar/zookeeper-nodes/";
+    private static final String KAFKA_CA_FILE_PATH = "/opt/kafka/cluster-ca-certs/ca.crt";
+    private static final String ZK_CA_FILE_PATH = "/etc/tls-sidecar/cluster-ca-certs/ca.crt";
+
+    public SystemtestCertManager() {}
+
+    private List<String> generateOpenSSLCommand(String server, String caFilePath, String hostname) {
+        return new ArrayList<>(asList("openssl",
+                "s_client",
+                "-connect", server,
+                "-showcerts",
+                "-CAfile", caFilePath,
+                "-verify_hostname", hostname
+        ));
+    }
+
+    public String openSSLCommandByResource(String podName, String hostname, String port, String customResource, String namespace) {
+        return openSSLCommandByResource(podName + "." + hostname + ":" + port,
+                podName + "." + hostname + "." + namespace + ".svc.cluster.local", podName, customResource, true);
+    }
+
+    public String openSSLCommandByResource(String server, String hostname, String podName, String customResource) {
+        return openSSLCommandByResource(server, hostname, podName, customResource, true);
+    }
+
+    public String openSSLCommandByResource(String server, String hostname, String podName, String customResource, boolean withCertAndKey) {
```

Comment:
What do you mean by custom resource? Custom resource `Kafka` contains also `Zookeeper` so I would suggest renaming it to component or something else...What do you think @Frawless ?

Reply:
How I said, naming is not my strong side, so component will be really better..